x-pack/plugins/infra/public/alerting/log_threshold/components/alert_details_app_section/components/logs_history_chart.tsx

```diff
@@ -0,0 +1,160 @@
+                defaultMessage: 'logs threshold alerts history',
+              })}
+            </h2>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiText size="s" color="subdued">
+            {i18n.translate('xpack.infra.logs.alertDetails.chartHistory.last30days', {
+              defaultMessage: 'Last 30 days',
+            })}
+          </EuiText>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer size="s" />
+      <EuiFlexGroup gutterSize="l">
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup gutterSize="xs" direction="column">
+            <EuiFlexItem grow={false}>
+              <EuiText color="danger">
+                <EuiTitle size="s">
+                  <h3>{triggeredAlertsData?.totalTriggeredAlerts || '-'}</h3>
+                </EuiTitle>
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiText size="s" color="subdued">
+                {i18n.translate('xpack.infra.logs.alertDetails.chartHistory.alertsTriggered', {
+                  defaultMessage: 'Alerts triggered',
+                })}
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <EuiFlexGroup gutterSize="xs" direction="column">
+          <EuiFlexItem grow={false}>
+            <EuiText>
+              <EuiTitle size="s">
+                <h3>
+                  {triggeredAlertsData?.avgTimeToRecoverUS
+                    ? convertTo({
+                        unit: 'minutes',
+                        microseconds: triggeredAlertsData?.avgTimeToRecoverUS,
+                        extended: true,
+                      }).formatted
+                    : '-'}
+                </h3>
+              </EuiTitle>
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiText size="s" color="subdued">
+              {i18n.translate('xpack.infra.logs.alertDetails.chartHistory.avgTimeToRecover', {
+                defaultMessage: 'Avg time to recover',
+              })}
+            </EuiText>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexGroup>
+      <EuiSpacer size="s" />
+      <CriterionPreview
+        annotations={[
+          <LineAnnotation
+            id="annotations"
+            key={'annotationsAlertHistory'}
+            domainType={AnnotationDomainType.XDomain}
+            dataValues={
+              triggeredAlertsData?.histogramTriggeredAlerts
+                .filter((annotation) => annotation.doc_count > 0)
+                .map((annotation) => {
+                  return {
+                    dataValue: annotation.key,
+                    header: String(annotation.doc_count),
+                    details: moment(annotation.key_as_string).format('yyyy-MM-DD'),
+                  };
+                }) || []
```

Comment:
**nit**: just for improved readability, could we move this computation into a descriptive variable before the return statement to keep clean the JSX?